src/Microsoft.ML.TensorFlow/TensorFlow/TensorflowUtils.cs

```diff
@@ -40,7 +40,10 @@ internal static Schema GetModelSchema(IExceptionContext ectx, TFGraph graph, str
 
                 // If the type is not supported in ML.NET then we cannot represent it as a column in an Schema.
                 // We also cannot output it with a TensorFlowTransform, so we skip it.
-                if (mlType == null)
+                // Furthermore, 'NoOp' operator usually has 'Unknown' type.
+                // But sometimes it can have valid type upon querying model for multiple times such as 'Float'.
+                // This behavior is inconsistent in TensorFlow runtime (#2156). Therefore, filtering all 'NoOp' operators in the model.
```

Comment:
(nit) - can you give a full URL for `(#2156)`? That way we know exactly which to look up.